crates/bevy_render/src/camera/camera.rs

```diff
@@ -349,15 +349,15 @@ pub fn camera_system<T: CameraProjection + Component>(
         })
         .collect();
 
-    let mut added_cameras = vec![];
+    let mut changed_cameras = vec![];
     for entity in &queries.p1() {
-        added_cameras.push(entity);
+        changed_cameras.push(entity);
     }
     for (entity, mut camera, mut camera_projection) in &mut queries.p0() {
         if camera
             .target
             .is_changed(&changed_window_ids, &changed_image_handles)
-            || added_cameras.contains(&entity)
+            || changed_cameras.contains(&entity)
```

Comment:
```suggestion
            || camera.is_changed()
```
Using `is_changed` on `camera` makes the second query unnecessary.